README.md

```diff
@@ -115,8 +115,8 @@ On Starknet, the deployment process is in two steps:
     ```
     Copy the declare Class Hash provided to use in the following step.
 3. Deploy:
```

Comment:
You can include a message explaining that you should pass the `owner` as the `CTOR_ARGS` argument

Reply:
Done!